### ISSUE TITLE
The argument type 'Container' can't be assigned to the parameter type '(BuildContext, String) → Widget'.

### ISSUE THREAD

**Issue by sagarshende23** [duytq94/flutter-chat-demo#11]
![screenshot 21](https://user-images.githubusercontent.com/43273993/53730416-dc667300-3e9d-11e9-9e9c-34fdb16a8ef2.png)

**Comment by duytq94** [comment]
I just check it and the project don't show any error like you, please check your version package, config, settings dart, flutter, and library. Finally clear cache and re-run, btw you should use Android Studio IDE to support better coding. Thanks.

**Comment by XSiyabonga** [comment]
Im having this exact problem
![image](https://user-images.githubusercontent.com/33574679/54356170-9bbbe600-4663-11e9-92f6-657e281513ad.png)

**Comment by duytq94** [comment]
Hi all, finally I got what you guys mean.
Your version of Cached network image is higher than the version when I build my project, so they change the syntax, temporary you just update your syntax like below:
![image](https://user-images.githubusercontent.com/24253421/54367857-b2832c80-46a5-11e9-9645-a42ec4f38cbb.png)
And more packages change since the day I built the project, so I'm updating the source.
Finally thanks for your report.
